qemu/tests/cfg/balloon_thp.cfg

```diff
@@ -0,0 +1,9 @@
+- balloon_thp:
+    type = balloon_thp
+    only Linux
+    balloon = balloon0
+    balloon_dev_devid_balloon0 = balloon0
+    balloon_dev_add_bus = 'yes'
+    fragement_dir = '/fragement'
+    cmd_make_fragement = 'mkdir -p ${fragement_dir} && mount -t tmpfs  none ${fragement_dir}'
+    cmd_make_fragement += ' && for i in `seq 10240`; do dd if=/dev/zero of=${fragement_dir}/$i count=1 bs=4k; done'
```

Comment:
Pls use 102400 instead, it will consume 400M memory, which make buddyinfo change evident.